### ISSUE TITLE
Decompiler missing from mac_x86_64

### ISSUE THREAD

**Issue by hxvu** [NationalSecurityAgency/ghidra#5979]
Installed from ghidra_10.4_PUBLIC. After a binary was imported, I went ahead and explore the binary but many errors popped up 
the results are the following errors:
-------
```
Cannot run program "/Users/hxv/ghidra_10.4_PUBLIC/Ghidra/Features/Decompiler/os/mac_x86_64/decompile": error=2, No such file or directory
java.io.IOException: Cannot run program "/Users/hxv/ghidra_10.4_PUBLIC/Ghidra/Features/Decompiler/os/mac_x86_64/decompile": error=2, No such file or directory
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1170)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1089)
	at java.base/java.lang.Runtime.exec(Runtime.java:681)
	at java.base/java.lang.Runtime.exec(Runtime.java:530)
	at ghidra.app.decompiler.DecompileProcess.setup(DecompileProcess.java:137)
	at ghidra.app.decompiler.DecompileProcess.registerProgram(DecompileProcess.java:433)
	at ghidra.app.decompiler.DecompInterface.initializeProcess(DecompInterface.java:284)
	at ghidra.app.decompiler.DecompInterface.openProgram(DecompInterface.java:381)
	at ghidra.app.decompiler.component.Decompiler.getDecompilerInterface(Decompiler.java:79)
	at ghidra.app.decompiler.component.Decompiler.decompile(Decompiler.java:48)
	at ghidra.app.decompiler.component.DecompilerManager.decompile(DecompilerManager.java:175)
	at ghidra.app.decompiler.component.DecompileRunnable.monitoredRun(DecompileRunnable.java:108)
	at ghidra.util.task.RunManager$RunnerJob.doExecute(RunManager.java:334)
	at ghidra.util.task.RunManager$RunnerJob.run(RunManager.java:309)
	at ghidra.util.worker.AbstractWorker$JobCallback.process(AbstractWorker.java:133)
	at ghidra.util.worker.AbstractWorker$JobCallback.process(AbstractWorker.java:123)
	at generic.concurrent.ConcurrentQ$CallbackCallable.call(ConcurrentQ.java:658)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at generic.concurrent.FutureTaskMonitor.run(FutureTaskMonitor.java:76)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.io.IOException: error=2, No such file or directory
	at java.base/java.lang.ProcessImpl.forkAndExec(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:295)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:225)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1126)
	... 21 more
```
---------------------------------------------------
Build Date: 2023-Sep-28 1301 EDT
Ghidra Version: 10.4
Java Home: /Users/hxv/jdk-21.0.1+12/Contents/Home
JVM Version: Eclipse Adoptium 21.0.1
OS: Mac OS X 12.6.6 x86_64
A clear and concise description of the bug.
-----------
**To Reproduce**
Follow install instructions
create a project 
import a binary
click on any symbol

**Expected behavior**
Decompiled anything

**Screenshots**
NA

**Attachments**
NA

**Environment (please complete the following information):**
 - OS: Darwin Kernel Version 21.6.0: Mon Apr 24 21:10:53 PDT 2023; root:xnu-8020.240.18.701.5~1/RELEASE_X86_64 x86_64
 - Java Version: 21
 - Ghidra Version: 10.4
 - Ghidra Origin: official GitHub distro

**Additional context**
NA

**Comment by ryanmkurtz** [comment]
Your decompiler executable most likely got quarantined by [Gatekeeper](https://en.wikipedia.org/wiki/Gatekeeper_(macOS)).  If so, this is a duplicate of #3663.  You can either go into your macOS `Privacy & Security` settings and unblock it, or build your own native components with the `support/buildNatives` script.  Note that building your own native components requires the correct build chain to be installed.  See the "Building Ghidra Native Components" section of the [Installation Guide](https://htmlpreview.github.io/?https://github.com/NationalSecurityAgency/ghidra/blob/master/GhidraDocs/InstallationGuide.html#Build) for more info.

**Comment by ryanmkurtz** [comment]
I suggest you redownload a fresh Ghidra release, check to see if the decompiler executable is where it should be, then launch Ghidra and see if it gets removed.

**Comment by hxvu** [comment]
Thank you for your help, it is a duplicate of https://github.com/NationalSecurityAgency/ghidra/issues/3663
